Library/Homebrew/linkage_checker.rb

```diff
@@ -291,7 +310,11 @@ def display_items(label, things, puts_output: true)
       end
     else
       things.sort.each do |item|
-        output += "\n  #{item}"
+        output += if item.is_a? Regexp
+          "\n #{item.inspect}"
```

Comment:
```suggestion
          "\n  #{item.inspect}"
```

unless the single space is intentional.